homeassistant/components/sense/binary_sensor.py

```diff
@@ -134,13 +108,18 @@ def icon(self):
     @property
     def device_class(self):
         """Return the device class of the binary sensor."""
-        return BIN_SENSOR_CLASS
+        return DEVICE_CLASS_POWER
 
     @property
     def should_poll(self):
         """Return the deviceshould not poll for updates."""
         return False
 
+    async def async_update(self):
+        """Get the latest data, update state."""
+        self._available = True
+        self._state = bool(self._sense_devices_data.get_device_by_id(self._id))
```

Comment:
This doesn't look like it would change the state to `False` if the device was using 0w

Reply:
@kbickar  The api doesn't appear report devices that are not using power so the get_device_by_id will be empty and false if using 0w

Reply:
For the above to be true I need to reset 

```
self._data_by_device = {}
```
on each update.

Done